shuffle/app/components/polls/statements/CreateStatementButton.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import type { FC } from "react";
+import React, { useCallback, useState } from "react";
+
+import { AnimatePresence } from "framer-motion";
+import { PlusCircle } from "lucide-react";
+
+import { useAmplitude } from "@/providers/AmplitudeProvider";
+
+import { CreateStatementDialog } from "./CreateStatementDialog";
+
+type Props = {
+  pollId: number;
+};
+
+export const CreateStatementButton: FC<Props> = ({ pollId }) => {
+  const { track } = useAmplitude();
+
+  const [isCreating, setIsCreating] = useState(false);
+
+  const closeDialog = useCallback(() => {
+    setIsCreating(false);
+  }, []);
+
+  const onNewStatement = useCallback(() => {
```

Comment:
Im not sure its worth wrapping simple things like this in useCallback. It might be worse performance wise than just having the function itself.